src/main/resources/application-test.yml

```
@@ -0,0 +1,17 @@
+spring:
+  profiles:
+    active: test
```

```suggestion
    active: test
  flyway:
    enabled: false
```

테스트 자체를 스킵하기보다는 해당 설정을 추가해보고 빌드 해보시겠어요~?

---

or 테스트 코드는 profile이 test일 경우는 h2 DB기반으로 작동되도록 profile 을 설정하면 될 것 같습니다 :)

---

넵 설정 변경해보겠습니다!